client-script-usage.md

```diff
@@ -23,6 +33,25 @@ Once the `sigridci.py` script is available within your CI environment, you can c
 [2] By default, Sigrid already excludes a number of files and directories from being analyzed. This includes third party libraries (for example `/node_modules/` for NPM libraries, build output (for example `/target/` for Maven builds), and generated code. Using the `--exclude` option will exclude additional files and directories *on top of* the ones that were already excluded.  
 [3] The `--pathprefix` option can be used in cases where your repository used a different directory structure from the one that is known to Sigrid. For example, you might have combined your back-end and front-end repositories within a single system in Sigrid, so that in Sigrid there are two top-level folders: `backend` and `frontend` containing the contents of your two repositories. However, you still want to get specific feedback for your front-end repository in Sigrid CI. In this case you would use `--pathprefix frontend` so that Sigrid CI knows the location of your repository within the larger directory structure. Note that you cannot use this option if you're already using `--publish`.  
 
+## Specifying a quality target
+
+Sigrid CI compares the quality of the new/changed code against the configured target quality level. The simplest and recommended way to configure the target is by using the `--targetquality` command line argument. This will check the overall quality maintainability rating against the target. 
+
+The advanced approach requires a configuration file named `sigrid.yaml` located in the root of the repository. This approach allows you to specify a target quality level for both the overall maintainability level and every system property. You can use a different target for different metrics, so this allows you to both have an overall target, but also be more strict (or more lenient) for some of the underlying system properties. Using the configuration file provides more flexibility and more control, but it also makes the feedback more complicated. In general, it is recommended to use the overall target quality level, and only start defining specific thresholds for specific system properties when there are structural quality issues that cannot be addressed otherwise.
```

Comment:
Maybe we can re-word as: 
"You can use a different target for different metrics..." -----> "You can use different targets per metric"?

Reply:
I suggest a middle ground in "You can use different targets for different metrics". I'm afraid some people coudl misinterpret "per metric".